src/main/scala/cromwell/binding/values/WdlFloat.scala

```diff
@@ -5,28 +5,68 @@ import cromwell.binding.types.WdlFloatType
 
 case class WdlFloat(value: Float) extends WdlPrimitive {
   val wdlType = WdlFloatType
-  def add(rhs: WdlValue): WdlValue = {
+  override def add(rhs: WdlValue): WdlValue = {
```

Comment:
i had made a mental note that these all needed overrides. now i can go back to forgetting that mental note.